packages/storage-plus/src/keys.rs

```diff
@@ -615,7 +626,7 @@ mod test {
         assert_eq!(pair.prefix(), vec![one.as_slice(), two.as_slice()]);
 
         let pair: (i8, &[u8]) = (123, b"random");
-        let one: Vec<u8> = vec![123];
+        let one: Vec<u8> = vec![123 + 128];
```

Comment:
This might look reasonable now, but for some time (and for other dev) this "magic" addition might... not add up (see what I did there?).
Either assign some const to `128` or add some comments maybe.

Reply:
You are right. Perhaps doing `vec![123 ^ 0x80];` is clearer?